api/src/services/metricsService.js

```diff
@@ -0,0 +1,24 @@
+const prom = require('prom-client');
+const { decimals } = require('../config/blockchains');
+const cacheService = require('./cacheService');
+const { convertSmallToBigCoins } = require('../lib/utils')
+
+const accountBalancesMetric = new prom.Gauge({
+  name: 'accounts_balances',
+  help: 'internal accounts balances amount',
```

Comment:
Let's remove `internal`. We are monitoring internal and external.

Reply:
resolved